### PR TITLE
Adding in-store support for orders.

### PR DESCRIPTION
**Pull request by crisfrincon**

### CODE REVIEW

File: lib/repositories/store.js [hunk 1]
```diff
@@ -31,24 +31,14 @@ class Store extends BaseRepository {
     };
   }
 
-  /**
-   * Returns the requestCustomBuilder service that corresponds to this repository
-   * @protected
-   * @param {string} key - store Key
-   * @returns {string} - Url to perform action on in-stores
-   */
-  requestCustomBuilder(key) {
```
Comment (xemayebenes) [inline lib/repositories/store.js:40]: why not instead of remove the method, you add a new param to set the carts or de orders?

Comment (crisfrincon) [reply lib/repositories/store.js:40]: Ok. Yes, I didn't see the value on that method, but yes, we can do that.

**Comment by crisfrincon** [comment]
Closing this PR as I will implement the logic with the Orders repository.

File: lib/repositories/store.js (hunk 1 above)
Comment (xemayebenes) [reply lib/repositories/store.js:40]: is just to be coherent with the rest of the repositories

**Pull request closed without merging**